Formula/container-diff.rb

```diff
@@ -0,0 +1,34 @@
+class ContainerDiff < Formula
+  desc "Diff your Docker containers"
+  homepage "https://github.com/GoogleCloudPlatform/container-diff"
+  url "https://github.com/GoogleCloudPlatform/container-diff/archive/v0.5.1.tar.gz"
+  sha256 "2de7635f485314ad8506222783c615620a4a7c32b0d39215df70f1b8964c871f"
+
+  head do
+    url "https://github.com/GoogleCloudPlatform/container-diff.git"
+  end
+
+  depends_on "go" => :build
+
+  def install
+    # The 0.5.1 release was tagged with an incorrect build version in the Makefile.
+    # It was fixed afterwards by https://github.com/GoogleCloudPlatform/container-diff/pull/142
```

Comment:
let's apply that as a patch instead to make sure this gets removed on the next update

Reply:
There's a new release upstream already, presumably with the fix in.

Reply:
even better

Reply:
The maintainer [was receptive to feedback](https://github.com/GoogleCloudPlatform/container-diff/issues/147).